code/modules/admin/gimmick/wizard_rings.dm

```diff
@@ -0,0 +1,279 @@
+/obj/item/clothing/gloves/ring/wizard
```

Comment:
should be done with ABSTRACT_TYPE, and every instance of childrentypesof be concretetypesof